signalflow/fake_backend.go

```diff
@@ -177,6 +177,26 @@ func (f *FakeBackend) handleMessage(ctx context.Context, message map[string]inte
 			resolutionMs = 1000
 		}
 
+		// use start and stop to control ending the fakebackend
+		var stopMs uint64
+		var startMs uint64
+		messageStopMs, _ := message["stop"].(float64)
+		if messageStopMs != 0.0 {
+			stopMs = uint64(messageStopMs)
+		}
+
+		if stopMs == 0 {
+			stopMs = uint64(time.Now().UnixNano() / (1000 * 1000))
+		}
+		messageStartMs, _ := message["start"].(float64)
+		if messageStartMs != 0.0 {
+			startMs = uint64(messageStartMs)
+		}
+
+		if stopMs == 0 {
```

Comment:
How could this ever be 0 since if it is 0 above on line 188 it gets set to the current time in ms?

Reply:
its a typo - fixed so startMs is now set to now in line with how the client default works